<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Téléchargements/ZpKyf3ZBgLFQTpMM1OeLzCjBV8xta9O0_0/logparser.jpg</normaloff>../../../../Téléchargements/ZpKyf3ZBgLFQTpMM1OeLzCjBV8xta9O0_0/logparser.jpg</iconset>
  </property>
  <widget class="QPushButton" name="pushButton_savePreferences">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>97</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_closePrefs">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>97</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Annuler</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_err404">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Erreurs 404</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_GBot">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Pages Crawlées par &amp;GoogleBot</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Bots">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Tous les &amp;bots et crawlers</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
